commit 7372fbad5c676267d12064b642f4d1558c3307b9 (refs/changes/37/2937/2)
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2017-11-27T16:12:39-05:00 (1 year, 10 months ago)
    
    NET-381 use triton-origin for booter/dhcpd
    TRITON-13 service configure.sh scripts are of inconsistent strictness
    NET-380 dhcpd service is only accidenlty enabled

diff --git a/Makefile b/Makefile
index dafc197..a681322 100644
--- a/Makefile
+++ b/Makefile
@@ -47,7 +47,7 @@ JSSTYLE_FLAGS = -o indent=4,doxygen,unparenthesized-return=0
 NODE_PREBUILT_VERSION=v0.10.48
 ifeq ($(shell uname -s),SunOS)
 	NODE_PREBUILT_TAG=zone
-	NODE_PREBUILT_IMAGE=de411e86-548d-11e4-a4b7-3bb60478632a
+	NODE_PREBUILT_IMAGE=18b094b0-eb01-11e5-80c1-175dac7ddf02
 endif
 
 
diff --git a/boot/configure.sh b/boot/configure.sh
index bfae9c7..783ff9c 100755
--- a/boot/configure.sh
+++ b/boot/configure.sh
@@ -7,18 +7,17 @@
 #
 
 #
-# Copyright (c) 2015, Joyent, Inc.
+# Copyright (c) 2017, Joyent, Inc.
 #
 
 export PS4='[\D{%FT%TZ}] ${BASH_SOURCE}:${LINENO}: ${FUNCNAME[0]:+${FUNCNAME[0]}(): }'
+set -o errexit
+set -o pipefail
 set -o xtrace
 
-echo "Importing dhcpd manifest"
+echo "Importing dhcpd manifest (default: enabled)"
 /usr/sbin/svccfg import /opt/smartdc/booter/smf/manifests/dhcpd.xml
 
-echo "Enabling dhcpd service"
-/usr/sbin/svcadm enable smartdc/site/dhcpd
-
 echo "Importing tftpd manifest"
 /usr/sbin/svccfg import /opt/smartdc/booter/smf/manifests/tftpd.xml
 
@@ -29,9 +28,9 @@ echo "Configuring nginx"
 cp /opt/smartdc/booter/etc/nginx.conf /opt/local/etc/nginx/nginx.conf
 
 echo "Importing nginx manifest"
-/usr/sbin/svccfg import /opt/local/share/smf/nginx/manifest.xml
+/usr/sbin/svccfg import /opt/local/lib/svc/manifest/nginx.xml
 
 echo "Enabling nginx service"
-/usr/sbin/svcadm enable network/nginx
+/usr/sbin/svcadm enable pkgsrc/nginx
 
 exit 0
